packages/react-components/react-nav-preview/src/components/Nav/Nav.types.ts

```diff
@@ -36,14 +36,14 @@ export type NavProps = ComponentProps<NavSlots> & {
    * The value of the currently selected navGroup.
    * Mutually exclusive with defaultSelectedValue.
    */
-  selectedValue?: NavGroupValue;
+  selectedValue?: NavCategoryItemValue;
```

Comment:
It looks like `NavCategoryItemValue` is (will be) used for things other than NavCategoryItem. Should it be named e.g. `NavItemValue` and shared across multiple different Nav*Item types?

Reply:
Yep, and moved the definition of NavItemValue into the context type to re-enforce how it should be generic.